### ISSUE TITLE
Map Local doesn't work when using non-standard HTTPS port

### ISSUE THREAD

**Issue by ianiv** [ProxymanApp/Proxyman#541]
### Steps to reproduce

1. Create a Map Local rule for POST requests matching `*/v1/*` and all subpaths
1. Run a server that listens to HTTPS requests on port 8443
1. Make a POST request to `https://yourserver:8443/v1/test`
1. Notice that the response does not contain the data from the Map Local rule
1. Change the server to listen on port 443
1. Repeat the same POST request, notice that the response contains the data from the Map Local rule

### Expected behavior
Map Local rules should apply regardless of the port used if the pattern matches the request

**Comment by NghiaTranUIT** [comment]
Thanks for the detailed report. Let me investigate and send the beta build soon 🙌

**Comment by ianiv** [comment]
I also noticed the same problem with breakpoints later in the day

**Comment by NghiaTranUIT** [comment]
You're right because they are the same code for URL matching 😄. I'm fixing it now 🌮

**Comment by NghiaTranUIT** [comment]
@ianiv  I'm not sure, but I couldn't reproduce your issue. It works well in the latest build 2.1.1

Here is my steps:
1. Setup a self-signed localhost HTTPS server. Map 0.0.0.0 to proxyman.dev in `/etc/host`. Thus, I can access via https://proxyman.dev:8443/v1/test
2. Open Proxyman. Run `curl -X POST -v https://proxyman.dev:8443/v1/test --proxy http://localhost:9090` and enable **SSL-Proxying** on this domain (Right Menu -> Enable SSL) and verify that I can see the HTTPS traffic
3. Create a new Map Local with rule `*/v1/*` with a default message
4. Repeat the curl and I'm able to see the Local file content

** Setup the same for the Breakpoint - Response and it's still working

<img width="2032" alt="Screen Shot 2020-07-04 at 08 56 27" src="https://user-images.githubusercontent.com/5878421/86503054-567d7380-bdd4-11ea-9d1a-d1a57dbd7ef9.png">


<img width="2032" alt="Screen Shot 2020-07-04 at 08 56 32" src="https://user-images.githubusercontent.com/5878421/86503238-364eb400-bdd6-11ea-9c2d-6fc17289405b.png">


----------------------
I suppose that you forgot to enable SSL-Proxying on your domain (Step 2). Please help me to verify that

**Comment by ianiv** [comment]
You are correct, it does work and I am not sure why it wasn't working before. I suspect it has to do with adding both a map local and breakpoint rule for the same pattern. I can't get that to work reliably.

**Comment by NghiaTranUIT** [comment]
Glad to know that it's working now. I will keep investigating this issue and bump it if I find 👍
